pkgs/top-level/aliases.nix

```diff
@@ -768,6 +768,11 @@ mapAliases ({
   noto-fonts-extra = noto-fonts; # Added 2023-04-08
   NSPlist = nsplist; # Added 2024-01-05
   nvidia-thrust = throw "nvidia-thrust has been removed because the project was deprecated; use cudaPackages.cuda_cccl";
+  nvtop = throw "nvtop has been renamed to nvtopPackages.full"; # Added 2024-02-25
+  nvtop-amd = throw "nvtop-amd has been renamed to nvtopPackages.amd"; # Added 2024-02-25
+  nvtop-nvidia = throw "nvtop-nvidia has been renamed to nvtopPackages.nvidia"; # Added 2024-02-25
+  nvtop-intel = throw "nvtop-intel has been renamed to nvtopPackages.intel"; # Added 2024-02-25
+  nvtop-msm = throw "nvtop-msm has been renamed to nvtopPackages.msm"; # Added 2024-02-25
```

Comment:
Hmm maybe this is too radical?

Reply:
Do you suggest to left old names as aliases for next release?

Reply:
> Do you suggest to left old names as aliases for next release?

Yes, though from my experience, such attributes are usually left behind for years, unless you'd remember to remove them when the branch-off of the next-next release happens.

Reply:
Replaced throw with lib.warn